aten/src/ATen/cpu/vec256/intrinsics.h

```diff
@@ -32,6 +32,9 @@
         (defined(__VEC__) || defined(__ALTIVEC__))
 /* XLC or GCC-compatible compiler, targeting PowerPC with VMX/VSX */
 #include <altivec.h>
+#undef bool
```

Comment:
Can we add similar comment here like the link you provided in https://github.com/pytorch/pytorch/issues/41261?
Probably just like the comment here:
https://github.com/eigenteam/eigen-git-mirror/blob/36b95962756c1fce8e29b1f8bc45967f30773c00/Eigen/src/Core/util/ConfigureVectorization.h#L366-L371